bridge/contracts/libraries/factory/AliceNetFactoryBase.sol

```diff
@@ -479,22 +477,18 @@ abstract contract AliceNetFactoryBase is DeterministicAddress, ProxyUpgrader {
      * @param isOk_ boolean false to cause revert
      */
     function _requireAuth(bool isOk_) internal pure {
-        require(
-            isOk_,
-            string(abi.encodePacked(AliceNetFactoryBaseErrorCodes.ALICENETFACTORYBASE_UNAUTHORIZED))
-        );
+        if (!isOk_) {
```

Comment:
yeah this is my biggest annoyance, because it doesnt work with `require` I have to reverse the logic and throw errors if its true, not as pretty but there's no other way it seems :(